autopush/tests/test_endpoint.py

```diff
@@ -1401,6 +1401,8 @@ def test_base_tags(self):
                                 host='example.com:8080')
         tags = self.reg.base_tags()
         eq_(tags, ['user_agent:test', 'host:example.com:8080'])
+        tags = self.reg.base_tags()
+        eq_(tags, ['user_agent:test', 'host:example.com:8080'])
```

Comment:
it's a lame test but it was failing before the fix, i could add a comment or just kill it